app/services/analytics.rb

```diff
@@ -191,6 +191,7 @@ def session_started_at
   OTP_SMS_INBOUND_MESSAGE_VALIDATION_FAILED = 'Twilio SMS Inbound Validation Failed'
   USER_MARKED_AUTHED = 'User marked authenticated'
   USER_REGISTRATION_AGENCY_HANDOFF_PAGE_VISIT = 'User registration: agency handoff visited'
+  USER_REGISTRATION_BACKUP_CODE_CONFIRMATION_SETUP_VISIT = 'Backup code confirmation setup visited'
```

Comment:
I don't want to hold up this PR too much, it's ben open for a while, so maybe split this out into a follow-up task, but could we add this as a new-style `analytics_events.rb` method instead?